app/models/user.rb

```diff
@@ -1,7 +1,11 @@
 # app/models/user.rb
 class User < ApplicationRecord
     validates :email, presence: true, uniqueness: true
+    validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: 'must be a valid email address' }
 
-    # Relationship with LmsCredential
+    # Associasions
     has_many :lms_credentials, dependent: :destroy
+    has_many :user_to_courses
+    has_many :lms_credentials
+    has_one :extensions
```

Comment:
Sorry if I caused any confusion in the ER diagram. I think a `user` should have many `extensions` this user processes as the last person, and an `extension` has one `user`. The `schema.rb` and migration files on the main branch are correct. 

I updated the [ER diagram](https://dbdiagram.io/d/ER-Diagram-65f0769eb1f3d4062cbd74e1), and the [ER diagram](https://dbdiagram.io/d/ER-Diagram-65f0769eb1f3d4062cbd74e1) is also included in the bookmarks on the team Slack channel.